zeppelin-web/src/app/tabledata/tabledata.js

```diff
@@ -60,7 +60,7 @@ export default class TableData extends Dataset {
         } else {
           let valueOfCol;
           if (!(col[0] === '0' || col[0] === '+' || col.length > float64MaxDigits)) {
-            if (!isNaN(valueOfCol = parseFloat(col)) && isFinite(col)) {
+            if (!isNaN(valueOfCol = col) && isFinite(col)) {
```

Comment:
Will changing this (removing `parseFloat`) matter in this case? If I understand it right, ideally it should not even come to this if.
Which makes me wonder what is `typeof col`, ideally that should be string, and if it was a string and `col.length` in this case should have been 19, which is greater than 16 (float64MaxDigits). 
Also if it is not a string then that a limitation of JS itself, as this is what I see on my browser console:
```
let col=6703273802681287331;
console.log(col);
6703273802681288000
```

Will try to repro this using JDBC interpreter,  what I used to test is the following paragraph:
```
%sh
echo -e "%table id\tage"
echo -e "6703273802681287331\t2"
```

Reply:
I think there is no problem. Because JS is to show that the real operations are run in the interpreter. This is also the simplest solution.